scripts/docker/zeppelin-interpreter/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM zeppelin-distribution:latest AS zeppelin-distribution
+
+FROM ubuntu:20.04
+
+LABEL maintainer="Apache Software Foundation <dev@zeppelin.apache.org>"
+
+ARG version="0.9.0-SNAPSHOT"
+
+ENV VERSION="${version}" \
+    Z_HOME="/opt/zeppelin"
+
+RUN set -ex && \
+    apt-get -y update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y openjdk-8-jre-headless wget && \
+    # Cleanup
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get autoclean && \
+    apt-get clean
+
+COPY --from=zeppelin-distribution /opt/zeppelin/bin ${Z_HOME}/bin
+COPY log4j.properties ${Z_HOME}/conf/
+COPY log4j_yarn_cluster.properties ${Z_HOME}/conf/
+# Decide:
+## 1) Build a huge image with all interpreters (default)
+COPY --from=zeppelin-distribution /opt/zeppelin/interpreter ${Z_HOME}/interpreter
+## 2) Build an image with only some interpreters
```

Comment:
we should create directory `${Z_HOME}/run` in this Dockerfile or in `interpreter.sh`. 

 `${Z_HOME}/run` is pid directory, when starting interpreter-server, the pid file would write to this dir. However, `interpreter.sh` does not create this directory, so if not create first, we would meet the following error:

`/opt/zeppelin/bin/interpreter.sh: line 305: /opt/zeppelin/run/zeppelin-interpreter-md-shared_process--md-dubwak-12320.pid: No such file or directory`

Reply:
Done in https://github.com/apache/zeppelin/pull/3859/files#diff-816be6e430bff3524979b549f7952db7R76